odoo/tools/pdf.py

```diff
@@ -59,7 +59,7 @@ class OdooPdfFileReader(PdfFileReader):
     # OVERRIDE of PdfFileReader to add the management of multiple embedded files.
 
     def getAttachments(self):
-        if "/Names" not in self.trailer["/Root"]:
+        if "/Names" not in self.trailer["/Root"] or "/EmbeddedFiles" not in self.trailer["/Root"]["/Names"] or "/Names" not in self.trailer["/Root"]["/Names"]["/EmbeddedFiles"]:
```

Comment:
`PyPDF2.generic.DictionaryObject` are dicts so we can use `d.get(key, default)`.
How about this more concise version? 

```suggestion
        if not self.trailer["/Root"].get("/Names", {}).get("/EmbeddedFiles", {}).get("/Names"):
```

Reply:
Much better, thanks !